apps/opik-backend/src/main/java/com/comet/opik/api/AlertEventType.java

```diff
@@ -16,7 +16,8 @@ public enum AlertEventType {
     PROMPT_CREATED("prompt:created"),
     PROMPT_COMMITTED("prompt:committed"),
     TRACE_GUARDRAILS_TRIGGERED("trace:guardrails_triggered"),
-    PROMPT_DELETED("prompt:deleted");
+    PROMPT_DELETED("prompt:deleted"),
+    EXPERIMENT_FINISHED("experiment:finished");
```

Comment:
Nit: to avoid a two line change next time
```suggestion
    EXPERIMENT_FINISHED("experiment:finished"),
    ;
```